client/src/components/refreshrs/Refreshr.js

```diff
@@ -82,10 +137,10 @@ function Refreshr(props) {
           InputLabelProps={{
             shrink: true
           }}
-          onChange={(e) => addRefreshrName(e.target.value)}
+          onChange={e => addRefreshrName(e.target.value)}
         />
-        <h4 className={props.classes.subheaders}>Review Text</h4>
-        <TextField
+        {/* <h4 className={props.classes.subheaders}>Review Text</h4> */}
+        {/* <TextField
```

Comment:
Looks like you're taking off fields for the question object, this will break how it's being send to the db, so you might need to mess w/ the migrations to match the new object you're passing up.